app/Http/Controllers/AuthController.php

```diff
@@ -20,33 +21,39 @@ public function register(RegisterRequest $request) {
         return response()->json([
             'token' => $token,
             'user' => Auth::user(),
-        ],200);
+        ], 200);
     }
-    
 
-    public function login(Request $request) {
-        $data = $request->validated();
-        $data['password'] = Hash::make($data['password']);
-        $user = User::create($data);
-        $token = Auth::login($user);
 
-        return response()->json([
+    public function login(Request $request)
+    {
+        $credentials = $request->only(['email', 'password']);
```

Comment:
Would be nice if this request also had validation for email and password like RegisterRequest